funfact/lang/interpreter/_ascii.py

```diff
@@ -17,9 +17,11 @@ def tensor(self, abstract, **kwargs):
         return str(abstract.symbol)
 
     @as_payload
-    def index(self, item, bound, **kwargs):
+    def index(self, item, bound, kron, **kwargs):
```

Comment:
Can an index be simultaneously `bound` and `kron`?

Reply:
Not really, since they represent conflicting types of operations. We can add a checker for this in the syntax validator.